tests/JestExt.test.ts

```diff
@@ -24,13 +24,13 @@ describe('JestExt', () => {
             jestVersionMajor: 17,
         }));
         new JestExt(projectWorkspace, channelStub, {});
-        expect(window.showErrorMessage).toBeCalledWith('This extension relies on Jest 18+ features, it will work, but the highlighting may not work correctly.');
+        expect(window.showErrorMessage).toBeCalledWith('This extension relies on Jest 20+ features, it continue to work, some features may not work correctly.');

```

Comment:
Wonder if we should just switch this to a snapshot.  That way we don't have the duplicated text and can just verify the snapshot.